tests/integration/test_charm.py

```diff
@@ -620,424 +696,365 @@ async def test_set_password_action(ops_test: OpsTest, test_pod):
+#                 "create",
+#                 "--username",
+#                 sa_name,
+#                 "--namespace",
+#                 namespace,
+#                 "--conf",
+#                 "spark.kubernetes.executor.request.cores=0.1",
+#                 "--conf",
+#                 "spark.executor.instances=3",
+#             ]
+#         ).returncode
+#         == 0
+#     )
+#
+#     logger.info("Changing configuration for kyuubi-k8s charm...")
+#     await ops_test.model.applications[APP_NAME].set_config({"service-account": sa_name})
+#
+#     logger.info("Waiting for kyuubi-k8s app to be idle...")
+#     await ops_test.model.wait_for_idle(
+#         apps=[APP_NAME],
+#         status="active",
+#         timeout=1000,
+#     )
+#
+#     logger.info("Running action 'get-jdbc-endpoint' on kyuubi-k8s unit...")
+#     kyuubi_unit = ops_test.model.applications[APP_NAME].units[0]
+#     action = await kyuubi_unit.run_action(
+#         action_name="get-jdbc-endpoint",
+#     )
+#     result = await action.wait()
+#
+#     jdbc_endpoint = result.results.get("endpoint")
+#     logger.info(f"JDBC endpoint: {jdbc_endpoint}")
+#
+#     logger.info("Testing JDBC endpoint by connecting with beeline with no credentials ...")
+#     process = subprocess.run(
+#         [
+#             "./tests/integration/test_jdbc_endpoint.sh",
+#             test_pod,
+#             jdbc_endpoint,
+#             "db_888",
+#             "table_888",
+#         ],
+#         capture_output=True,
+#     )
+#     print("========== test_jdbc_endpoint.sh STDOUT =================")
+#     print(process.stdout.decode())
+#     print("========== test_jdbc_endpoint.sh STDERR =================")
+#     print(process.stderr.decode())
+#     logger.info(f"JDBC endpoint test returned with status {process.returncode}")
+#     assert process.returncode == 0
+#
+#     # Check exactly 3 executor pods were created.
+#     list_pods_process = subprocess.run(
+#         ["kubectl", "get", "pods", "-n", namespace, "--sort-by", ".metadata.creationTimestamp"],
+#         capture_output=True,
+#     )
+#
+#     assert list_pods_process.returncode == 0
+#
+#     pods_list = list_pods_process.stdout.decode().splitlines()
+#
+#     driver_pod_name = ""
+#     executor_pod_names = []
+#
+#     # Last 4 pods in the list are of interest,
+#     # one is the driver and 3 should be executor pods
+#     for pod in pods_list[-4:]:
+#         name = pod.split()[0]
+#         if "driver" in name:
+#             driver_pod_name = name
+#         else:
+#             executor_pod_names.append(name)
+#
+#     expected_executor_pod_names = [
+#         driver_pod_name.replace("driver", "exec-1"),
+#         driver_pod_name.replace("driver", "exec-2"),
+#         driver_pod_name.replace("driver", "exec-3"),
+#     ]
+#
+#     assert len(executor_pod_names) == len(expected_executor_pod_names)
+#     assert set(executor_pod_names) == set(expected_executor_pod_names)
+
```

Comment:
[IMPORTANT] Don't forget to uncomment client tests once they are fixed on the pipeline.

Reply:
All tests are enabled again. Related issues to be discussed on the dedicated PR https://github.com/canonical/kyuubi-k8s-operator/pull/39